docs/features.md

```diff
@@ -0,0 +1,297 @@
+
+Default settings:
+
+```bash
+"EVENTING ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"eventing.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **KUBECONFIG_ID** – is used to configure URL to which Busola sends a request to download a kubeconfig file. If we add kubeconfigID={your ID} to the Busola URL then Busola tries to download the kubeconfig from the {kubeconfigUrl}/{yourID} and, if succedded, will add a cluster.
+  You can introduce the whole address in the kubeconfigUrl which will also be read by Busola.
+
+Default settings:
+
+```bash
+"KUBECONFIG_ID": {
+ "isEnabled": true,
+ "config": {
+   "kubeconfigUrl": "https://kyma-env-broker.cp.dev.kyma.cloud.sap/kubeconfig"
+ }
+},
+```
+
+- **LEGAL_LINKS** – is used to show or hide legal links. In config you can find the unchangeable keys (you cannot use legalDisclosure instead of legal-disclosure). The keys include the default link, which takes you to the default address, and a link that depends on your chosen language, for example:
+
+```bash
+"LEGAL_LINKS":{
+  "config": {
+    “legal-disclosure”: {
+      "default":" https://www.sap.com/corporate/en/legal/impressum.html",
+      "de":" https://www.sap.com/corporate/de/legal/impressum.html"
+    }
+  }
+},
+```
+
+A link under the given key will be selected based on your language code (de, en, pl, etc.), If the code is not available, the default link is used.
+
+- **OBSERVABILITY** – is used to render a few nodes in the navigation. The label shows the name of the given service. The path is used by Busola during the bootstrapping. Busola sends a request to the cluster address + the path and the cluster must return the virtualService object. If the object is found you receive an address to which the node in the navigation leads.
+
+Defualt settings:
+
+```bash
+"OBSERVABILITY": {
+ "isEnabled": true,
+ "config": {
+   "links": [
+     {
+       "label": "Grafana",
+       "path": "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/monitoring-grafana"
+     },
+     {
+       "label": "Kiali",
+       "path": "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/kiali"
+     },
+     {
+       "label": "Tracing",
+       "path": "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/tracing"
+     }
+   ]
+ }
+}
+```
+
+- **PROTECTED_RESOURCES** – is used to block the edit and delete functions based on the determined rules. If the resource meets the rule requirements, the resource becomes protected and cannot be edited/deleted.
+  Each resource must have a match field, which is the list of key-value pairs. The proper rule description is when the definition given in the key matches the value.
+
+  Optionally you can provide a message parameter, which displays a simple message, or messageSrc, which is a yaml path where the message to be displayed is included. If neither message nor messageSrc is provided, a generic message will be used.
```

Comment:
What is generic message?

Reply:
That is a good question :) Maybe Kuba can help us with that.

Reply:
It looks like there's no generic message - it should be `common.protected-resource`, but this key is not present in translations...

Reply:
I think we should document it and add the generic message.